### ISSUE TITLE
Add support for checking Vue files with vls

### ISSUE THREAD

**Issue by Dcryfm** [w0rp/ale#1296]
## Information

**neovim version**: 0.2.2

Operating System: elementary

I saw this in the help document:

```txt
LSP linters can be used in combination with any other linter, and
will automatically connect to LSP servers when needed.
```

then I searched for the information in the #517

and then I try to do this;

```vim
let g:ale_linters = {
  \'vue': ['vls'],
\}
```

but it didn't work, I have installed vue-language-server.
What should I do? Thanks.

**Comment by w0rp** [comment]
Someone needs to define a linter for it. That one doesn't exist.

**Comment by Dcryfm** [comment]
@w0rp So, like css-language-server, html-language-server, json-language-server and so on can't be used yet?

**Comment by Dcryfm** [comment]
I want to report the diagnostic information of the language server with the other linter tool's diagnostic information by ale.

**Comment by w0rp** [comment]
That can be used as soon as someone defines a linter for them, like everything else. Look at the `ale_linters` directory.

**Comment by w0rp** [comment]
Implemented in #1699
